CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# History
+
+## 1.1 (2017/03/19)
+
+- Added message bus tags
+- Moved semantic descriptions to `semantic_conventions.md` and `semantic_conventions.yaml`
+- Added OpenTracing governance information to `project_organization.md`
```

Comment:
@wu-sheng good suggestion, though the list is not finalized yet. I still have 11 days, right? :)

(Arguably I should have left this line off since it's not part of the spec, but I was trying to summarize `git log` and figured it couldn't hurt)